pkg/sinks/udsink/sink.go

```diff
@@ -141,8 +126,3 @@ func (s *UserDefinedSink) Stop() {
 func (s *UserDefinedSink) ForceStop() {
 	s.isdf.ForceStop()
 }
-
-// IsHealthy checks if the udsink sidecar is healthy.
```

Comment:
why is this removed?

Reply:
https://github.com/numaproj/numaflow/blob/cb1a38b4dc0ab9bf4ab4a789e3da59cf38564ea0/pkg/sinks/udsink/udsink_grpc.go#L49 we already have that method inside UDSgRPCBasedUDSink which will be used by the metrics server.